pyecharts/utils/__init__.py

```diff
@@ -4,6 +4,9 @@
 import codecs
 import os
 
+__all__ = ['get_resource_dir', 'write_utf8_html_file', 'to_css_length',
+           'merge_js_dependencies']
```

Comment:
with this, you could allow `_add` to be outside merge_js_dependencies.

Reply:
`_add` references `dependencies` and `fist_items` by pointer (Not value reference)